<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="eventFileTxt">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browseEventBtn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>200</y>
      <width>93</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="chooseEventFileLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Choose an event file:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newModelBtn">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>490</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Create new model</string>
    </property>
   </widget>
   <widget class="QLabel" name="selectModelLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>240</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select model:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="classifyEventBtn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>310</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Classify event</string>
    </property>
   </widget>
   <widget class="QLabel" name="headLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gabriola</family>
      <pointsize>29</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Classify Event</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="modelFileTxt">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="backBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt; Back</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>801</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="eventNameTxt">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="eventNameLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Event name:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browseModelBtn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>270</y>
      <width>93</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>430</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="errorLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>360</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>430</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Classification Progress:</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>801</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QMdiArea" name="topBlock">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>431</width>
      <height>291</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="background">
     <brush brushstyle="Dense5Pattern">
      <color alpha="100">
       <red>110</red>
       <green>116</green>
       <blue>160</blue>
      </color>
     </brush>
    </property>
   </widget>
   <widget class="QMdiArea" name="middleBlock">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>410</y>
      <width>801</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="background">
     <brush brushstyle="Dense5Pattern">
      <color alpha="100">
       <red>110</red>
       <green>116</green>
       <blue>160</blue>
      </color>
     </brush>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>490</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>View Previous Classifications</string>
    </property>
   </widget>
   <widget class="QMdiArea" name="bottomBlock">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>791</width>
      <height>61</height>
     </rect>
    </property>
    <property name="background">
     <brush brushstyle="Dense5Pattern">
      <color alpha="100">
       <red>110</red>
       <green>116</green>
       <blue>160</blue>
      </color>
     </brush>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>789</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
